customizations/views.py

```diff
@@ -134,76 +183,52 @@ class ViewCreateCustomization(LoginRequiredMixin, FormView):
         else:
             # if we keep links['status'] the logo is a required field
             if form.is_valid():
-                name = request.POST.get('name')
-                message = request.POST.get('message')
-                select_design_template = request.POST.get('select_design_template')
-                message_ticket = request.POST.get('message_ticket')
-                show_event_sequence = request.POST.get('show_event_sequence') == 'on'
-                show_ticket_type_sequence = request.POST.get('show_ticket_type_sequence') == 'on'
-                hide_ticket_type_price = request.POST.get('hide_ticket_type_price') == 'on'
-                footer_description = request.POST.get('footer_description')
-                double_ticket = request.POST.get('double_ticket') == 'on'
-                pdf_ticket_attach = request.POST.get('pdf_ticket_attach')
-                template = get_object_or_404(
-                    BaseTicketTemplate,
-                    pk=select_design_template)
-                ticket = TicketTemplate.objects.create(
-                    select_design_template=template,
-                    message_ticket=message_ticket,
-                    show_event_sequence=show_event_sequence,
-                    show_ticket_type_sequence=show_ticket_type_sequence,
-                    hide_ticket_type_price=hide_ticket_type_price,
-                    footer_description=footer_description,
-                    double_ticket=double_ticket,
-                )
-                custom_email = CustomEmail.objects.create(
-                    message=message,
-                    logo=links['dropbox'],
-                    logo_local=links['local'],
-                    logo_path=links['path'],
-                    logo_name=links['name'],
-                    logo_url=links['dropbox'],
-                    image_partner=partner_links['dropbox'],
-                    image_partner_local=partner_links['local'],
-                    image_partner_path=partner_links['path'],
-                    image_partner_name=partner_links['name'],
-                    image_partner_url=partner_links['dropbox'],
-                )
-                Customization.objects.create(
-                    user=self.request.user,
-                    ticket_template=ticket,
-                    custom_email=custom_email,
-                    name=name,
-                    pdf_ticket_attach=pdf_ticket_attach,
-                )
-                if not UserWebhook.objects.filter(user=request.user).exists():
-                    token = get_token(request.user)
-                    webhook_id = create_webhook(token)
-                    UserWebhook.objects.create(
-                        user=self.request.user,
-                        webhook_id=webhook_id,
+                if compose_customization(request):
+                    return HttpResponseRedirect('/')
+                else:
```

Comment:
This can be simplified, a good idea is have all the `validate()` statements in the beginning of the `post` method, and encapsulates the logic in a single block, after call the action in the domain and catch possible exceptions:

```
# Validations
if not form.is_valid:
    form.error = ...
    render(...

# Domain action
try:
    compose_customization({})
catch PossibleException:
    handle()
```